src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -202,6 +202,18 @@ public async Task DispatchEvent(string eventDescriptorJson, string eventArgs)
             }
         }
 
+        internal void OnRenderCompleted(long renderId, string errorMessageOrNull)
```

Comment:
Can be `public`, since the type itself is already `internal`

Reply:
Will change

Reply:
done